### PR TITLE
Bug 1157149 - added a secondary count indicating the number of filter…

### PR DESCRIPTION
**Pull request by kkrivera**
…ed unclassified failures

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/1908)

### CODE REVIEW

**Comment by kkrivera** [comment]
@camd please review.

File: ui/partials/main/thWatchedRepoNavPanel.html [hunk 1]
```diff
@@ -8,11 +8,19 @@
...
+                         'btn-view-nav': getAllUnclassifiedFailureCount(repoName)===0}"
               ng-click="toggleUnclassifiedFailures()">
           <span id="unclassified-failure-count">
-            {{ getUnclassifiedFailureCount(repoName) }}</span> unclassified
+            {{ getAllUnclassifiedFailureCount(repoName) }}</span> unclassified
+        </span>
+
+        <!--Filtered Unclassified Failures Button-->
+        <span class="navbar-badge badge"
+              title="Reflects the currently unclassified failures which are being filtered"
```
Comment (camd) [inline ui/partials/main/thWatchedRepoNavPanel.html]: Let's go with the wording of: "Reflects the unclassified failures which pass the current filters"

File: ui/css/treeherder-navbar.css [hunk 2]
```diff
@@ -374,3 +374,7 @@ fieldset[disabled] .btn-view-nav-closed.active {
     text-decoration: line-through;
   }
 }
+
+.navbar-badge.badge * {
+  cursor: default;
+}
```
Comment (camd) [inline ui/css/treeherder-navbar.css]: Please go ahead and add the newline at the end of the file.

**Comment by kkrivera** [comment]
@camd I made the changes. please review again.

**Pull request merged**